### DISCUSSION TITLE
MouseVR Godot Project for "Waveshare 240x240"

### DISCUSSION THREAD

**Discussion by ryojimo** [sn-lab/MouseGoggles#25] [Show and tell]
Hi,
I appreciate this good system.
[TT108RGN10A] was Out of stock, so that I used [Waveshare 240x240].
I'm beginer of Godot and I can not find which part I should change.
Is it possible to upload Godot project for "Waveshare 240x240" ?

![image](https://github.com/sn-lab/MouseGoggles/assets/31120847/63c2fe81-a1ca-4cbc-bb04-c4bcbe3e894f)

[TT108RGN10A]: https://www.alibaba.com/product-detail/Small-IPS-Round-LCD-Module-1_62550674738.html
[Waveshare 240x240]: https://www.waveshare.com/product/1.28inch-lcd-module.htm

**Comment by ryojimo** [comment]
I found it by myself, thanks.
The changes from **MouseVR Godot Project V1.0** are as the following.
[diff.zip](https://github.com/sn-lab/MouseGoggles/files/13930706/diff.zip)

**Reply by misaacson01** [reply]
Glad you got it worked out! You might not need any of this by now, but FYI I have also just finished updating the repository with files that can be used to build MouseGoggles (v1.5) with these Waveshare displays, including new 3D prints, a custom PCB, a new Godot project folder, and assembly/installation instructions. These files can be located [Versions/1.5](https://github.com/sn-lab/MouseGoggles/tree/main/Versions/1.5).

**Comment by ryojimo** [comment]
I appreciate your update, [Versions/1.5/MouseVR Godot Project V1.5].
I works on my Raspberrypi4, thanks.

Just in case, let me share about the changes in [fbcp-ili9341] for Waveshare 240x240, too.

* [fbcp-ili9341/config.h]
```
-#define GPIO_TFT_DATA_CONTROL 6
-#define GPIO_TFT_RESET_PIN 22
+#define GPIO_TFT_DATA_CONTROL 25
+#define GPIO_TFT_RESET_PIN 27
```

[Versions/1.5/MouseVR Godot Project V1.5]: https://github.com/sn-lab/MouseGoggles/tree/main/Versions/1.5/MouseVR%20Godot%20Project%20V1.5
[fbcp-ili9341]: https://github.com/sn-lab/MouseGoggles/tree/main/fbcp-ili9341
[fbcp-ili9341/config.h]: https://github.com/sn-lab/MouseGoggles/blob/main/fbcp-ili9341/config.h
